src/coreclr/utilcode/posterror.cpp

```diff
@@ -123,10 +123,19 @@ static HRESULT FormatRuntimeErrorVA(
         hr = UtilLoadStringRC(LOWORD(hrRpt), rcBuf, ARRAY_SIZE(rcBuf), true);
         if (hr == S_OK)
         {
-            MAKE_UTF8PTR_FROMWIDE(rcUtf8, rcBuf);
-            _vsnprintf_s(msgBufUtf8, ARRAY_SIZE(msgBufUtf8), _TRUNCATE, rcUtf8, marker);
-            MAKE_WIDEPTR_FROMUTF8(msgBuf, msgBufUtf8);
-            wcscpy_s(rcMsg, cchMsg, msgBuf);
+            EX_TRY
+            {
+                MAKE_UTF8PTR_FROMWIDE(rcUtf8, rcBuf);
```

Comment:
A nit - there is also a MAKE_UTF8PTR_FROMWIDE_NOTHROW / MAKE_WIDEPTR_FROMUTF8_NOTHROW in case you'd like to use that.

Reply:
Sure. It does make the logic more verbose I think. Let me know what you think.